docs/sources/panels/panel-library.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Library panel"
+weight = 400
++++
+
+# Library panel
+
+Library panels allow users to create reusable panels that aren’t tied to any one dashboard. As a result, any changes made to one instance of the library panel affecting all other instances.
```

Comment:
We need to more fully explain what library panels are. This can be done later.

Reply:
Agree